### PR TITLE
test: add tests

### PR DESCRIPTION
**Pull request by luckydrq**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change
add test cases for watcher plugin

### CODE REVIEW

File: test/development.test.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+require('should');
+const mm = require('egg-mock');
+const fs = require('fs');
+const request = require('supertest');
+const utils = require('./utils');
+const file_path1 = utils.getFilePath('apps/watcher-development-app/tmp.txt');
+const file_path2 = utils.getFilePath('apps/watcher-development-app/tmp/tmp.txt');
+const file_path1_agent = utils.getFilePath('apps/watcher-development-app/tmp-agent.txt');
+
+describe('test/development.test.js', function() {
```
Comment (fengmk2) [inline test/development.test.js]: arrow function

**Comment by fengmk2** [comment]
## [Current coverage][cc-pull] is **57.53%**


> No coverage report found for **master** at 1740523.

> Powered by [Codecov](https://codecov.io?src=pr). Last updated by [1740523...af0c1c3][cc-compare]
[cc-base-branch]: https://codecov.io/gh/eggjs/egg-watcher/branch/master?src=pr
[cc-compare]: https://codecov.io/gh/eggjs/egg-watcher/compare/17405238501e1311627a931c5af0ff1f47c4ead1...af0c1c3da91a91a3c84412ae75e9a5211165ed8b?src=pr
[cc-pull]: https://codecov.io/gh/eggjs/egg-watcher/pull/1?src=pr

**Comment by fengmk2** [comment]
覆盖率这么低？

**Comment by luckydrq** [comment]
本地 tnpm run ci 是这个结果：
=============================== Coverage summary ===============================
Statements   : 86.49% ( 96/111 )
Branches     : 68.42% ( 26/38 )
Functions    : 83.33% ( 20/24 )
Lines        : 89.62% ( 95/106 )

**Comment by luckydrq** [comment]
windows 下居然会报这个错
```
1) test/development.test.js "before each" hook for "should agent watcher work":
     Error: ENOTEMPTY: directory not empty, rmdir 'C:\projects\egg-watcher\test\fixtures\apps\watcher-development-app\logs\watcher-app'
      at Error (native)
      at Object.fs.rmdirSync (fs.js:758:18)
      at rmkidsSync (C:\projects\egg-watcher\node_modules\.npminstall\rimraf\2.5.3\rimraf\rimraf.js:342:11)
      at rmdirSync (C:\projects\egg-watcher\node_modules\.npminstall\rimraf\2.5.3\rimraf\rimraf.js:332:7)
      at rimrafSync (C:\projects\egg-watcher\node_modules\.npminstall\rimraf\2.5.3\rimraf\rimraf.js:303:9)
      at C:\projects\egg-watcher\node_modules\.npminstall\rimraf\2.5.3\rimraf\rimraf.js:340:5
      at Array.forEach (native)
      at rmkidsSync (C:\projects\egg-watcher\node_modules\.npminstall\rimraf\2.5.3\rimraf\rimraf.js:339:26)
      at rmdirSync (C:\projects\egg-watcher\node_modules\.npminstall\rimraf\2.5.3\rimraf\rimraf.js:332:7)
      at Function.rimrafSync [as sync] (C:\projects\egg-watcher\node_modules\.npminstall\rimraf\2.5.3\rimraf\rimraf.js:303:9)
      at Function.Coffee.constructor.module.exports [as cluster] (C:\projects\egg-watcher\node_modules\.npminstall\egg-mock\0.0.2\egg-mock\lib\cluster.js:167:12)
      at Context.<anonymous> (C:\projects\egg-watcher\test\development.test.js:16:14)
      at Context._fn (C:\projects\egg-watcher\node_modules\.npminstall\thunk-mocha\1.0.3\thunk-mocha\index.js:27:59)
```

File: package.json [hunk 2]
```diff
@@ -24,9 +24,11 @@
     "wt": "~1.1.0"
   },
   "devDependencies": {
-    "autod": "^2.6.1",
-    "egg-bin": "^1.0.2",
-    "egg-ci": "^1.0.2",
+    "autod": "2",
+    "egg": "^0.0.5",
```
Comment (fengmk2) [inline package.json]: 改成 *

**Comment by fengmk2** [comment]
+1

**Comment by luckydrq** [comment]
rebase了，发1.0.0 ?

**Comment by fengmk2** [comment]
@luckydrq 发吧

**Comment by luckydrq** [comment]
egg-watcher@1.0.0

**Comment by luckydrq** [comment]
cc @atian25 @jtyjty99999

**Pull request merged**